### PR TITLE
feat: show taskbar progress bar during installation

### PR DESCRIPTION
**Pull request by ErickSharp**
## Summary of Changes

Shows a taskbar progress bar during installation.

## Screenshots (if necessary)

image
image

## Additional context

Discord username (if different from GitHub): Erick [Z-6]#6484

### CODE REVIEW

File: src/renderer/components/AircraftSection/index.tsx [hunk 1]
```diff
@@ -176,6 +177,15 @@ const index: React.FC<TransferredProps> = (props: AircraftSectionProps) => {
         }
     }, [selectedTrack, installedTrack]);
 
+    useEffect(() => {
+        if (download && isDownloading) {
+            console.log(download.progress);
```
Comment (FoxtrotSierra6829) [inline src/renderer/components/AircraftSection/index.tsx:182]: please wrap this console log into some context.

**Review by FoxtrotSierra6829 — CHANGES_REQUESTED** [review]

**Comment by FoxtrotSierra6829** [comment]
Also please add a changelog entry, as this is something which is noticeable to the user

**Comment by ErickSharp** [comment]
> Also please add a changelog entry, as this is something which is noticeable to the user

You want me to edit the changelog.md file in the .github directory, correct? Just double checking as it seems there aren't many entries.

**Comment by FoxtrotSierra6829** [comment]
> 
> 
> > Also please add a changelog entry, as this is something which is noticeable to the user
> 
> You want me to edit the changelog.md file in the .github directory, correct? Just double checking as it seems there aren't many entries.

Yes

**Review by FoxtrotSierra6829 — APPROVED** [review]

**Pull request merged**